zanata.org redesign

kathryngo opened the pull request:
# Zanata.org redesign
## Bootstrap 4 alpha v6
Static site
### Homepage
Go to....
`
/index.html 
`
for the new homepage design
![home](https://user-images.githubusercontent.com/18179401/28950640-0186873a-7909-11e7-87e8-013f07870cb6.png)

The website has been created in a new folder and does not replace the current index.html file. This is because this is just the homepage and other pages need to be created before going live. Shall complete this next tech debt sprint.

QA: please pay attention to how this page handles different screen sizes.

I think it would be best to add all the pages on this one branch so I can link them and test navigation.

### About
Go to....
`
/about.html 
`
for the new About design

<img width="1440" alt="about-zorg" src="https://user-images.githubusercontent.com/18179401/29200926-cc7689fc-7e9c-11e7-9ed4-9036e9ed0895.png">


### Release Notes
Go to....
`
/release-notes.html 
`
for the Release Notes page design

![releasenotes](https://user-images.githubusercontent.com/18179401/28950646-0c5dd35c-7909-11e7-9312-d7bcebd50f93.png)

### Blog

Go to....
`
/blog.html
`
for the Blog page design - static only. Will use Jekyll blog in future.

![blog](https://user-images.githubusercontent.com/18179401/28950648-12020d5a-7909-11e7-9692-5e28cdaab914.png)


### Communities

Go to....
`
/communities.html
`
for the Communities page design

<img width="1440" alt="communities-zorg" src="https://user-images.githubusercontent.com/18179401/29200927-d34a0cae-7e9c-11e7-8159-bb41d0b3ab84.png">


### Instances

Go to....
`
/instances.html
`
for the Instances page design (was communities)

<img width="1440" alt="instances" src="https://user-images.githubusercontent.com/18179401/29200930-e4510548-7e9c-11e7-87c1-087c9d59c5e8.png">


<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/zanata/zanata.github.io/35)
<!-- Reviewable:end -->

aeng commented:
@kathryngo Don't think we still need the /redesign, probably can move everything up to root dir.

kathryngo commented:
@aeng Ok, shall give that a go :)

kathryngo commented:
Moved and tested @aeng

kathryngo commented:
@djansen-redhat please review/QA this when you get the chance

davidmason commented:
Review status: 0 of 71 files reviewed at latest revision, 2 unresolved discussions.

---

*[about.html, line 9 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1KgZawnQMZl0BJ0KD:-Ks1KgZawnQMZl0BJ0KE:brsh4kc) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L9)):*
> ```HTML
>           name="viewport">
>     <meta content="About Zanata translation software" name="description">
>     <meta content="The Zanata team" name="Red Hat">
> ```

Stylistic preference: meta `name` before `content` (since we put keys before values everywhere else). Same with link (rel before href). Is this some convention I'm not aware of putting them the other way?

---

*[about.html, line 20 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1MIqJ-qZGysxFiT3Q:-Ks1MIqJ-qZGysxFiT3R:b-8igcj9) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L20)):*
> ```HTML
> 
> <body class="">
> <nav class="navbar navbar-toggleable-md navbar-inverse fixed-top bg-inverse">
> ```

This nav section is repeated almost identically across multiple pages. Is there a script or something to generate these? If not it is going to be difficult to maintain, so we should set up a decent static site generator before the next time we update this.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

aeng commented:
Review status: 0 of 71 files reviewed at latest revision, 2 unresolved discussions.

---

*[about.html, line 20 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1MIqJ-qZGysxFiT3Q:-Ks1OjkUpD_eFVUEsQdw:bo3yrn4) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L20)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

This nav section is repeated almost identically across multiple pages. Is there a script or something to generate these? If not it is going to be difficult to maintain, so we should set up a decent static site generator before the next time we update this.
</blockquote></details>

I dont't think this should be a blocker for this PR. This is just a static html page. Try not to complicate things.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

kathryngo commented:
Review status: 0 of 71 files reviewed at latest revision, 2 unresolved discussions.

---

*[about.html, line 9 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1KgZawnQMZl0BJ0KD:-Ks1doQ8SL-913uOEh9W:bhrz4rj) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L9)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

Stylistic preference: meta `name` before `content` (since we put keys before values everywhere else). Same with link (rel before href). Is this some convention I'm not aware of putting them the other way?
</blockquote></details>

Shall fix this, thanks

---

*[about.html, line 20 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1MIqJ-qZGysxFiT3Q:-Ks1dqZ-d66JHMAOR0aP:b-abb2t3) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L20)):*
<details><summary><i>Previously, aeng (Alex Eng) wrote…</i></summary><blockquote>

I dont't think this should be a blocker for this PR. This is just a static html page. Try not to complicate things.
</blockquote></details>

Yep I agree, lets keep this simple. Maintaining this site will not be too difficult as static pages.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

davidmason commented:
Reviewed 5 of 141 files at r2, 4 of 10 files at r3.
Review status: 5 of 71 files reviewed at latest revision, 16 unresolved discussions.

---

*[about.html, line 20 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1MIqJ-qZGysxFiT3Q:-Ks1P7XMWg5sc-UsGUR7:b9948po) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L20)):*
<details><summary><i>Previously, aeng (Alex Eng) wrote…</i></summary><blockquote>

I dont't think this should be a blocker for this PR. This is just a static html page. Try not to complicate things.
</blockquote></details>

Yeah, not for this PR since it is already done.

This does add a lot of time to code review though, and will be more work to update (not to mention more chance of accidental inconsistency). There are plenty of static site generators out there, and we just want fairly basic templating, so it should be pretty easy to set that up next time there are going to be any changes that would need copy-paste across all these html files.

---

*[about.html, line 62 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1TMszBbQeBmlBR3D7:-Ks1TMszBbQeBmlBR3D8:b-kygo7s) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L62)):*
> ```HTML
>             <a href="http://github.com/zanata/zanata-platform" role="button"
>                class="btn btn-link my-2 my-sm-0">
>                 <i class="fa fa-github-square fa-2x"></i>
> ```

This link needs a text fallback of some kind, for accessibility and just people who don't know the github logo.

---

*[about.html, line 74 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1UV3e9NONurkJKbwZ:-Ks1UV3e9NONurkJKbw_:bcqeu0p) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L74)):*
> ```HTML
>         documentation
>         into different languages.</p>
>     <p>Zanata is written in Java and uses the following modern web
> ```

"Java and JavaScript" (we've added a lot of JavaScript since this was originally written, so it has reached the point where we should include it).

---

*[about.html, line 107 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1Wg-4af6Nzu5rNycU:-Ks1Wg-4af6Nzu5rNycV:btx8lm9) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L107)):*
> ```HTML
>             <a href="hibernate.org">
>                 <img class="img-thumbnail"
>                      src="assets/hibernate.png" alt="hibernate"/>
> ```

Use caps for the names in `alt`: Java, Wildfly, GWT, React, Hibernate

---

*[about.html, line 111 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1_axhdqk_ZS4wKjVJ:-Ks1_axhdqk_ZS4wKjVK:b-6pt9ze) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L111)):*
> ```HTML
>         </div>
>         <div class="col">
>         </div>
> ```

We should include JavaScript (since we mention Java), and we should include Redux (since we mention GWT).

 - logo for JavaScript: https://github.com/voodootikigod/logo.js
 - Official Redux logos (a few styles to choose from): https://github.com/reactjs/redux/tree/master/logo

---

*[about.html, line 114 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1X8YO8bIXJXomfEme:-Ks1X8YO8bIXJXomfEmf:b-et70vr) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L114)):*
> ```HTML
>     </div>
> 
>     <p>It also presents a <a
> ```

"Zanata presents a..."

---

*[about.html, line 123 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1XHfBH-kngzNNAK0A:-Ks1XHfBH-kngzNNAK0B:b-7nurms) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L123)):*
> ```HTML
>         from a Zanata server using a Maven plugin or a
>         <a href="http://docs.zanata.org/en/release/client/">command-line
>             client.</a>
> ```

"You can upload to and download from Zanata using a Maven plugin, command-line client, or through the website."

---

*[about.html, line 144 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1XYZffm0s6804kGak:-Ks1XYZffm0s6804kGal:b7p4118) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L144)):*
> ```HTML
> </div>
> 
> <footer class="footer">
> ```

Footer will be easier to update if we use templating. This one doesn't change at all between pages (I assume), so even a really basic script would work.

---

*[about.html, line 169 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1XysJkLjqBNBOglMN:-Ks1XysJkLjqBNBOglMO:b-bkh672) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L169)):*
> ```HTML
>             <li><a href="mailing-lists.html">Mailing Lists</a>
>             </li>
>             <li><a href="issues.html">Issues on Jira</a>
> ```

Link text implies that this link will point to Jira, but it points to issues.html.

---

*[index.html, line 72 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1bUSbdm4qzY2PYcXI:-Ks1bUSbdm4qzY2PYcXJ:b2hzb7f) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/index.html#L72)):*
The platform is stronger with more people on the same server (because there is more in the server TM). This sounds a bit too much like encouraging or challenging technical people to run their own server. Could we make it something like:

> Can't share your text with the community? You can run your own Zanata server

---

*[index.html, line 77 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1duXDpn130-b1JtMX:-Ks1duXDpn130-b1JtMY:b-t6ihvq) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/index.html#L77)):*
> ```HTML
>                             Download Zanata</a> and read the
>                         <a href="http://docs.zanata.org/en/release/user-guide/system-admin/configuration/installation/">
>                             System Admin Guide</a>
> ```

These links both go to the same place. Is that an accidental wrong URL, or should we combine these to a single link?

---

*[index.html, line 89 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1eBU_rhfVmQR85Yhd:-Ks1eBU_rhfVmQR85Yhe:bvhznvk) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/index.html#L89)):*
> ```HTML
>                         <a href="">Download Zanata</a> and read the <a href="">System
>                             Admin
>                             Guide</a>
> ```

No URLs for narrow screens :iphone:  :frowning_face: 

---

*[index.html, line 117 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1eqAt_FXMWNMIcFQ2:-Ks1eqAt_FXMWNMIcFQ3:blwquqz) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/index.html#L117)):*
> ```HTML
>                         <li>No command line</li>
>                         <li>No source conflicts</li>
>                         <li>Translation memory shows history of similar translations to re-use
> ```

"reuse"

---

*[index.html, line 135 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1fL455eUXLGnAZCuH:-Ks1fL455eUXLGnAZCuI:byd6jxz) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/index.html#L135)):*
> ```HTML
>                             <a href="http://docs.zanata.org/en/release/user-guide/projects/project-types/#supported-types">
>                                 Currently supported</a>: Publican DocBook, LibreOffice
>                             and plaintext
> ```

This list is out of date. It should include HTML, subtitles, InDesign Markup Language.

There is a link on that supported-types page that goes to http://docs.zanata.org/en/release/user-guide/documents/raw-documents/ where most of the supported formats are shown.

---

*[index.html, line 152 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1gHsGXA1-7mlQNcDL:-Ks1gHsGXA1-7mlQNcDM:b-q6vxhy) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/index.html#L152)):*
> ```HTML
>                             <a href="http://docs.zanata.org/en/release/user-guide/projects/project-types/#supported-types">
>                                 Currently supported</a>: GNU Gettext-based
>                             PO files, Java Properties, XLIFF
> ```

Should include Qt Linguist, Mozilla DTD, JSON.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

davidmason commented:
Review status: 5 of 71 files reviewed at latest revision, 20 unresolved discussions.

---

*[instances.html, line 74 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1nubnk6rpSoAYwCjJ:-Ks1nubnk6rpSoAYwCjK:b-r084kn) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/instances.html#L74)):*
> ```HTML
>                 <a href="https://translate.openstack.org">
>                     <img class="img-thumbnail"
>                          src="assets/openstack.png" alt="openstack"
> ```

OpenStack

---

*[instances.html, line 84 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1nxO2BIqZ6Jm2fkbE:-Ks1nxO2BIqZ6Jm2fkbF:bt0c0pq) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/instances.html#L84)):*
> ```HTML
>                 <a href="https://fedora.zanata.org">
>                     <img class="img-thumbnail"
>                          src="assets/fedora.png" alt="fedora"/>
> ```

Fedora

---

*[instances.html, line 94 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1o1yDDXSQgdxgL5Si:-Ks1o1yDDXSQgdxgL5Sj:bebo59g) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/instances.html#L94)):*
> ```HTML
>             <p class="text-center"><a href="https://translate.jboss.org">
>                 <img class="img-thumbnail" src="assets/wildfly.png"
>                      alt="wildfly"/>
> ```

Wildfly

---

*[instances.html, line 106 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1qZPUXun96x3DDjSm:-Ks1qZPUXun96x3DDjSn:b-wlwo7o) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/instances.html#L106)):*
> ```HTML
>                     <img class="img-thumbnail"
>                          src="assets/zanata-text.png"
>                          alt="zanata"/></a><br/>
> ```

Zanata

---

*[instances.html, line 114 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1qidito71TnML_Fz3:-Ks1qidito71TnML_Fz4:bcqjsb5) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/instances.html#L114)):*
This could sound a little friendlier or something (not sure, gut instinct here) with wording more like:

> Disclaimer: Each instance is managed by its respective community. The Zanata team strive to provide a great platform, but have no direct control over the server update schedule, user management, project management, or hosted content (including spam).

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

davidmason commented:
Reviewed 2 of 141 files at r2, 2 of 10 files at r3.
Review status: 7 of 71 files reviewed at latest revision, 22 unresolved discussions.

---

*[issues.html, line 66 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks22DJlZZFCYkAg2lCw:-Ks22DJlZZFCYkAg2lCx:bc7lu2s) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/issues.html#L66)):*
> https://zanata.atlassian.net/secure/RapidBoard.jspa?rapidView=1

This is not visible without a JIRA account. We should probably move it below to the list of things you can do when you have an account.

---

*[issues.html, line 111 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1wgveWNTSqqUGxZD0:-Ks1wgveWNTSqqUGxZD1:b-4vx3gc) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/issues.html#L111)):*
> ```HTML
>             </li>
>             <li>
>                 <a href="issues.html">Issues on Jira</a>
> ```

Should this have a highlighted style since it is selected?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

kathryngo commented:
Review status: 7 of 71 files reviewed at latest revision, 22 unresolved discussions.

---

*[about.html, line 20 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1MIqJ-qZGysxFiT3Q:-Ks2PYGCRQbtXbURBgtS:bsdiruf) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L20)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

Yeah, not for this PR since it is already done.

This does add a lot of time to code review though, and will be more work to update (not to mention more chance of accidental inconsistency). There are plenty of static site generators out there, and we just want fairly basic templating, so it should be pretty easy to set that up next time there are going to be any changes that would need copy-paste across all these html files.
</blockquote></details>

It would add more development time than code review time at this point. I made sure that things are consistent across all files.  It really is not that much effort to update a link across 6 or so static pages. 

---

*[about.html, line 123 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1XHfBH-kngzNNAK0A:-Ks2REhn6cvKsELoMcwU:b-1ctm1u) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L123)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

"You can upload to and download from Zanata using a Maven plugin, command-line client, or through the website."
</blockquote></details>

Much better

---

*[about.html, line 144 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1XYZffm0s6804kGak:-Ks2Q2q3O3MvMv8KSWa_:bv1otdh) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L144)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

Footer will be easier to update if we use templating. This one doesn't change at all between pages (I assume), so even a really basic script would work.
</blockquote></details>

I realise this, but its outside the scope of this PR.

---

*[about.html, line 169 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1XysJkLjqBNBOglMN:-Ks2RCB8bNmkgtl0gnIt:b-u35eq7) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L169)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

Link text implies that this link will point to Jira, but it points to issues.html.
</blockquote></details>

Understood, shall fix.

---

*[index.html, line 72 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1bUSbdm4qzY2PYcXI:-Ks2RAKqQ4Fd7bmnRuOl:b-zcjvte) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/index.html#L72)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

The platform is stronger with more people on the same server (because there is more in the server TM). This sounds a bit too much like encouraging or challenging technical people to run their own server. Could we make it something like:

> Can't share your text with the community? You can run your own Zanata server
</blockquote></details>

Thanks

---

*[index.html, line 77 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1duXDpn130-b1JtMX:-Ks2QT-g2SSYcvaCSF9P:bh8o77k) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/index.html#L77)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

These links both go to the same place. Is that an accidental wrong URL, or should we combine these to a single link?
</blockquote></details>

Accidental, shall fix.

---

*[index.html, line 89 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1eBU_rhfVmQR85Yhd:-Ks2Qb6N9r69m16YMFSI:b-xtkxcz) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/index.html#L89)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

No URLs for narrow screens :iphone:  :frowning_face: 
</blockquote></details>

What do you mean? The links are still there on mobile layouts.

---

*[index.html, line 135 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1fL455eUXLGnAZCuH:-Ks2QhVk7z-M115MG0iN:bl0i2up) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/index.html#L135)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

This list is out of date. It should include HTML, subtitles, InDesign Markup Language.

There is a link on that supported-types page that goes to http://docs.zanata.org/en/release/user-guide/documents/raw-documents/ where most of the supported formats are shown.
</blockquote></details>

Thanks, been trying to get an updated list from the team, much appreciated.

---

*[instances.html, line 114 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1qidito71TnML_Fz3:-Ks2QnAau852co2zkReq:bnfyukd) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/instances.html#L114)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

This could sound a little friendlier or something (not sure, gut instinct here) with wording more like:

> Disclaimer: Each instance is managed by its respective community. The Zanata team strive to provide a great platform, but have no direct control over the server update schedule, user management, project management, or hosted content (including spam).
</blockquote></details>

Thats great, thanks.

---

*[issues.html, line 66 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks22DJlZZFCYkAg2lCw:-Ks2QqIzrfVSHYucudDW:b-4wpzlu) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/issues.html#L66)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

> https://zanata.atlassian.net/secure/RapidBoard.jspa?rapidView=1

This is not visible without a JIRA account. We should probably move it below to the list of things you can do when you have an account.
</blockquote></details>

Good eye, thanks.

---

*[issues.html, line 111 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1wgveWNTSqqUGxZD0:-Ks2QyIlACqelX_0jlbY:b-b2pa52) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/issues.html#L111)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

Should this have a highlighted style since it is selected?
</blockquote></details>

Guess so even though it isn't part of the main site structure and the link appears below the content.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

kathryngo commented:
Review status: 7 of 71 files reviewed at latest revision, 23 unresolved discussions.

---

*[about.html, line 9 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1KgZawnQMZl0BJ0KD:-Ks5tjc3oRqbfOHtHpK4:b-896fix) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L9)):*
<details><summary><i>Previously, kathryngo wrote…</i></summary><blockquote>

Shall fix this, thanks
</blockquote></details>

Done.

---

*[about.html, line 62 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1TMszBbQeBmlBR3D7:-Ks5w6QyHoCb1QV753jW:b-896fix) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L62)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

This link needs a text fallback of some kind, for accessibility and just people who don't know the github logo.
</blockquote></details>

Done.

---

*[about.html, line 74 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1UV3e9NONurkJKbwZ:-Ks5wEn7OxH2uVqAWuLD:b-896fix) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L74)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

"Java and JavaScript" (we've added a lot of JavaScript since this was originally written, so it has reached the point where we should include it).
</blockquote></details>

Done.

---

*[about.html, line 107 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1Wg-4af6Nzu5rNycU:-Ks5wR93SSTay7LdrC83:b-896fix) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L107)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

Use caps for the names in `alt`: Java, Wildfly, GWT, React, Hibernate
</blockquote></details>

Done.

---

*[about.html, line 111 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1_axhdqk_ZS4wKjVJ:-Ks5z5VPXJxTcHE7oigV:b-896fix) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L111)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

We should include JavaScript (since we mention Java), and we should include Redux (since we mention GWT).

 - logo for JavaScript: https://github.com/voodootikigod/logo.js
 - Official Redux logos (a few styles to choose from): https://github.com/reactjs/redux/tree/master/logo


</blockquote></details>

Done.

---

*[about.html, line 114 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1X8YO8bIXJXomfEme:-Ks5wTsQeYaTKkR1MyLp:b-896fix) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L114)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

"Zanata presents a..."
</blockquote></details>

Done.

---

*[about.html, line 123 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1XHfBH-kngzNNAK0A:-Ks5wlz4cMXaRtqeWURI:b-896fix) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L123)):*
<details><summary><i>Previously, kathryngo wrote…</i></summary><blockquote>

Much better
</blockquote></details>

Done.

---

*[about.html, line 169 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1XysJkLjqBNBOglMN:-Ks5z_3D-jrI3Q598Fqq:b-896fix) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L169)):*
<details><summary><i>Previously, kathryngo wrote…</i></summary><blockquote>

Understood, shall fix.
</blockquote></details>

Done.

---

*[index.html, line 72 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1bUSbdm4qzY2PYcXI:-Ks6-YDbBokAfuYtS6OH:b-896fix) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/index.html#L72)):*
<details><summary><i>Previously, kathryngo wrote…</i></summary><blockquote>

Thanks
</blockquote></details>

Done.

---

*[index.html, line 77 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1duXDpn130-b1JtMX:-Ks6-eRg_-CZ18Gokhhi:b-896fix) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/index.html#L77)):*
<details><summary><i>Previously, kathryngo wrote…</i></summary><blockquote>

Accidental, shall fix.
</blockquote></details>

Done.

---

*[index.html, line 117 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1eqAt_FXMWNMIcFQ2:-Ks6-nlURKdXjX8f2KCO:b-896fix) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/index.html#L117)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

"reuse"
</blockquote></details>

Done.

---

*[index.html, line 135 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1fL455eUXLGnAZCuH:-Ks61-ya2y3pQiq9RASG:b-896fix) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/index.html#L135)):*
<details><summary><i>Previously, kathryngo wrote…</i></summary><blockquote>

Thanks, been trying to get an updated list from the team, much appreciated.
</blockquote></details>

Done.

---

*[index.html, line 152 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1gHsGXA1-7mlQNcDL:-Ks61CouuTTUGO4uxFx8:b-896fix) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/index.html#L152)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

Should include Qt Linguist, Mozilla DTD, JSON.
</blockquote></details>

Done.

---

*[instances.html, line 74 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1nubnk6rpSoAYwCjJ:-Ks61DqMN2CU4zA1L_Gb:b-896fix) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/instances.html#L74)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

OpenStack
</blockquote></details>

Done.

---

*[instances.html, line 84 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1nxO2BIqZ6Jm2fkbE:-Ks61EHngO3mfikkMglr:b-896fix) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/instances.html#L84)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

Fedora
</blockquote></details>

Done.

---

*[instances.html, line 94 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1o1yDDXSQgdxgL5Si:-Ks61Els32_urkYwpn4R:b-896fix) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/instances.html#L94)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

Wildfly
</blockquote></details>

Done.

---

*[instances.html, line 106 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1qZPUXun96x3DDjSm:-Ks61FGOEKK1XlORDVKQ:b-896fix) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/instances.html#L106)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

Zanata
</blockquote></details>

Done.

---

*[instances.html, line 114 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1qidito71TnML_Fz3:-Ks61Khma_DxlW495uvJ:b-896fix) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/instances.html#L114)):*
<details><summary><i>Previously, kathryngo wrote…</i></summary><blockquote>

Thats great, thanks.
</blockquote></details>

Done.

---

*[issues.html, line 66 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks22DJlZZFCYkAg2lCw:-Ks61UCH19rmm0VAA4Uv:b-896fix) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/issues.html#L66)):*
<details><summary><i>Previously, kathryngo wrote…</i></summary><blockquote>

Good eye, thanks.
</blockquote></details>

Done.

---

*[issues.html, line 111 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1wgveWNTSqqUGxZD0:-Ks61fu0cPuuwPFJJ2L7:b-896fix) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/issues.html#L111)):*
<details><summary><i>Previously, kathryngo wrote…</i></summary><blockquote>

Guess so even though it isn't part of the main site structure and the link appears below the content.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

davidmason commented:
Reviewed 42 of 141 files at r2, 1 of 10 files at r3, 8 of 11 files at r4.
Review status: 28 of 73 files reviewed at latest revision, 5 unresolved discussions.

---

*[about.html, line 123 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1XHfBH-kngzNNAK0A:-Ks6iBz7ezuE32TRR_Ii:b-23gh7l) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L123)):*
<details><summary><i>Previously, kathryngo wrote…</i></summary><blockquote>

Done.
</blockquote></details>

Yay, I'm helping :)

---

*[about.html, line 144 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1XYZffm0s6804kGak:-Ks6iGyO8Fggvd7Pp3qt:b672zut) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/about.html#L144)):*
<details><summary><i>Previously, kathryngo wrote…</i></summary><blockquote>

I realise this, but its outside the scope of this PR.
</blockquote></details>

Agreed :thumbsup: 

We haven't been changing it very often anyway (every few years or something?) so doesn't matter much.

---

*[index.html, line 89 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1eBU_rhfVmQR85Yhd:-Ks6jXsZBDEgSLzUEcC6:b-dgkalb) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/index.html#L89)):*
<details><summary><i>Previously, kathryngo wrote…</i></summary><blockquote>

What do you mean? The links are still there on mobile layouts.
</blockquote></details>

Oh, just going by the class `hidden-lg-up` on the div so I assumed it was this div on smaller screens.

You added the URL for the guide now, but still has: `<a href="http">translate.zanata.org</a>`

---

*[css/bootstrap.min.css, line 0 at r4](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks6m0pE1QwBviLg4us2:-Ks6m0pE1QwBviLg4us3:bnvw9uc) ([raw file](https://github.com/zanata/zanata.github.io/blob/b066939969a02b6db2ab46ced0fcd53cacc2a778/css/bootstrap.min.css#L0)):*
Is this the standard bootstrap css (downloaded from them?). If so, it would probably be best to use one from a CDN if available.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

davidmason commented:
Reviewed 86 of 141 files at r2, 3 of 11 files at r4.
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

davidmason commented:
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---

*[css/bootstrap.min.css, line  at r4](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks6m0pE1QwBviLg4us2:-Ks6psE7_bzAgWWHHCd6:bv80wop) ([raw file](https://github.com/zanata/zanata.github.io/blob/b066939969a02b6db2ab46ced0fcd53cacc2a778/css/bootstrap.min.css#L)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

Is this the standard bootstrap css (downloaded from them?). If so, it would probably be best to use one from a CDN if available.
</blockquote></details>

Looks like this is the one they recommend on the bootstrap page, and they have 4.0.0-beta on it: https://www.bootstrapcdn.com/

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

kathryngo commented:
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---

*[index.html, line 89 at r3](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks1eBU_rhfVmQR85Yhd:-Ks7tGUjC662gm6sY0SE:b-oieriw) ([raw file](https://github.com/zanata/zanata.github.io/blob/fe107870df73fa28e1b1beb822471e91e881712a/index.html#L89)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

Oh, just going by the class `hidden-lg-up` on the div so I assumed it was this div on smaller screens.

You added the URL for the guide now, but still has: `<a href="http">translate.zanata.org</a>`
</blockquote></details>

Shall fix the links.

---

*[css/bootstrap.min.css, line  at r4](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks6m0pE1QwBviLg4us2:-Ks7tJ2H8JWVKNhB7LrL:b-sraug8) ([raw file](https://github.com/zanata/zanata.github.io/blob/b066939969a02b6db2ab46ced0fcd53cacc2a778/css/bootstrap.min.css#L)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

Looks like this is the one they recommend on the bootstrap page, and they have 4.0.0-beta on it: https://www.bootstrapcdn.com/
</blockquote></details>

Cheers, shall use 3 until 4 is out of beta

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

kathryngo commented:
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---

*[css/bootstrap.min.css, line  at r4](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks6m0pE1QwBviLg4us2:-Ks7vkJIBjXalqcHCrNK:b-yzfg02) ([raw file](https://github.com/zanata/zanata.github.io/blob/b066939969a02b6db2ab46ced0fcd53cacc2a778/css/bootstrap.min.css#L)):*
<details><summary><i>Previously, kathryngo wrote…</i></summary><blockquote>

Cheers, shall use 3 until 4 is out of beta
</blockquote></details>

Oops, disregard,  Using 4. :)

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

kathryngo commented:
Changes made @davidmason

davidmason commented:
Reviewed 7 of 141 files at r2, 8 of 8 files at r5.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[css/bootstrap.min.css, line  at r4](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks6m0pE1QwBviLg4us2:-KsBPh5ISirfHIG-qWrz:b8moa0t) ([raw file](https://github.com/zanata/zanata.github.io/blob/b066939969a02b6db2ab46ced0fcd53cacc2a778/css/bootstrap.min.css#L)):*
<details><summary><i>Previously, kathryngo wrote…</i></summary><blockquote>

Oops, disregard,  Using 4. :)
</blockquote></details>

Since we're getting bootstrap from CDN, we can delete this one can't we?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

davidmason commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> just delete that bootstrap.min.js file if none of the html files are pointing to it. +(Passed review)  -(Awaiting review) 

---

Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-:-KsBQELLG978ypasp-1t:b-apo2gj)*
<!-- Sent from Reviewable.io -->

kathryngo commented:
There is no bootstrap.min.js file, just the css file. I am only using the cdn link for the js file. 

---

Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-:-KsBSznmGyiRIAmeKxml:bync1az)*
<!-- Sent from Reviewable.io -->

kathryngo commented:
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[css/bootstrap.min.css, line  at r4](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks6m0pE1QwBviLg4us2:-KsBU26S7ONpPZqBEAzU:b-tqakmr) ([raw file](https://github.com/zanata/zanata.github.io/blob/b066939969a02b6db2ab46ced0fcd53cacc2a778/css/bootstrap.min.css#L)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

Since we're getting bootstrap from CDN, we can delete this one can't we?
</blockquote></details>

We are using JS from CDN, bootstrap css is local. Should I make it consistent and use both from CDN? I just wanted to decrease page load times.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

davidmason commented:
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[css/bootstrap.min.css, line  at r4](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks6m0pE1QwBviLg4us2:-KsBUDamz30TwwgmJzaZ:b92shq7) ([raw file](https://github.com/zanata/zanata.github.io/blob/b066939969a02b6db2ab46ced0fcd53cacc2a778/css/bootstrap.min.css#L)):*
<details><summary><i>Previously, kathryngo wrote…</i></summary><blockquote>

We are using JS from CDN, bootstrap css is local. Should I make it consistent and use both from CDN? I just wanted to decrease page load times.
</blockquote></details>

We should use CDN whenever we can. CDN load time is lower for users - that's the main reason CDNs exist.

The CDN servers are located around the globe so they can serve content from somewhere physically closer to the user, which makes it faster. There is also a good chance they would already have the CDN files cached from another site (very likely with something common like bootstrap) so they don't need to download anything.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

kathryngo commented:
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[css/bootstrap.min.css, line  at r4](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks6m0pE1QwBviLg4us2:-KsBWP5qqWmHaQ7x7Lji:b-97jgbo) ([raw file](https://github.com/zanata/zanata.github.io/blob/b066939969a02b6db2ab46ced0fcd53cacc2a778/css/bootstrap.min.css#L)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

We should use CDN whenever we can. CDN load time is lower for users - that's the main reason CDNs exist.

The CDN servers are located around the globe so they can serve content from somewhere physically closer to the user, which makes it faster. There is also a good chance they would already have the CDN files cached from another site (very likely with something common like bootstrap) so they don't need to download anything.
</blockquote></details>

Good to know. Shall change the css link and remove the file

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

kathryngo commented:
*[css/bootstrap.min.css, line  at r4](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks6m0pE1QwBviLg4us2:-KsBYHDfTMULZNovPgoC:bhh1jk0) ([raw file](https://github.com/zanata/zanata.github.io/blob/b066939969a02b6db2ab46ced0fcd53cacc2a778/css/bootstrap.min.css#L)):*
<details><summary><i>Previously, kathryngo wrote…</i></summary><blockquote>

Good to know. Shall change the css link and remove the file
</blockquote></details>

Umm, when I changed to the beta from alpha 6 it broke a heap of the layout, mostly the media queries. For the moment can we just use my local css until I work out what is going on there. The mobile menu is active all the time and the jumbotron layout on the homepage is really messed up. Since this site needs to go out ASAP, I will add a note of this to Jira to fix at a later date.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

davidmason commented:
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[css/bootstrap.min.css, line  at r4](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks6m0pE1QwBviLg4us2:-KsBZ3HR4Bfg1YEFmDri:b-j970lr) ([raw file](https://github.com/zanata/zanata.github.io/blob/b066939969a02b6db2ab46ced0fcd53cacc2a778/css/bootstrap.min.css#L)):*
<details><summary><i>Previously, kathryngo wrote…</i></summary><blockquote>

Umm, when I changed to the beta from alpha 6 it broke a heap of the layout, mostly the media queries. For the moment can we just use my local css until I work out what is going on there. The mobile menu is active all the time and the jumbotron layout on the homepage is really messed up. Since this site needs to go out ASAP, I will add a note of this to Jira to fix at a later date.
</blockquote></details>

This has alpha.6 (despite the name, just scroll down): https://www.bootstrapcdn.com/beta/

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

davidmason commented:
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[css/bootstrap.min.css, line  at r4](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-Ks6m0pE1QwBviLg4us2:-KsBZR6Byv5ZyD6i-NbS:b-ckop00) ([raw file](https://github.com/zanata/zanata.github.io/blob/b066939969a02b6db2ab46ced0fcd53cacc2a778/css/bootstrap.min.css#L)):*
<details><summary><i>Previously, davidmason (David Mason) wrote…</i></summary><blockquote>

This has alpha.6 (despite the name, just scroll down): https://www.bootstrapcdn.com/beta/
</blockquote></details>

Better do 4.0.0-alpha.6 for both the JS and CSS, since they're meant to work together.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

davidmason commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> :thumbsup: :thumbsup: :thumbsup: :thumbsup: :thumbsup: 

---

Reviewed 8 of 141 files at r2, 9 of 9 files at r6.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-:-KsBc0OSGLb4QE0YLmoq:b39zscp)*
<!-- Sent from Reviewable.io -->

kathryngo commented:
@davidmason sweet, will just need you to resolve the review in github.

aeng commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-:-KsBfLWsV9pvZjfVU9Tj:bnfp4nl)*
<!-- Sent from Reviewable.io -->

djansen-redhat commented:
- [ ] Drop down menu not showing correctly on about page, narrow view
![screenshot from 2017-09-01 13-56-06](https://user-images.githubusercontent.com/4597195/29954728-55f27002-8f1d-11e7-87e4-7f281088cc89.png)

kathryngo commented:
Thanks @djansen-redhat will check out what's going wrong there

kathryngo commented:
@djansen-redhat can't replicate. works for me. maybe try and and make sure that the page doesn't time out? it may have just not loaded the icon. apologies if you checked that already, just is the most obvious reason for this.

kathryngo commented:
Review status: 72 of 75 files reviewed at latest revision, 1 unresolved discussion.

---

*[a discussion](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-KsvjTHDSNC5aORBrS1O:-KsvjTHDSNC5aORBrS1P:bi65a5k) (no related file):*
@aeng I have added some files to this PR so that the install scripts work correctly. I did this under the advice and guidance of Damian. You will need to check these files for this to pass review when you have the chance please.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

djansen-redhat commented:
I've worked it out.
It's because the folders are still being served up, eg. $server/blog/ and $server/blog.html are two different pages (index.html).

aeng commented:
Reviewed 1 of 3 files at r7.
Review status: 73 of 75 files reviewed at latest revision, 1 unresolved discussion.

---

*[a discussion](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-KsvjTHDSNC5aORBrS1O:-KsxYg0hY3c-BCz-wn59:bnq41og) (no related file):*
<details><summary><i>Previously, kathryngo wrote…</i></summary><blockquote>

@aeng I have added some files to this PR so that the install scripts work correctly. I did this under the advice and guidance of Damian. You will need to check these files for this to pass review when you have the chance please.
</blockquote></details>

Which files you're referring to?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

aeng commented:
Reviewed 2 of 3 files at r7.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

djansen-redhat commented:
I've found what's up with this PR.
There are two conflicting methods of generating the site. The site is built using the assets directory, eg. `assets/blog/index.html` and now also the flat site with `./blog.html`. There is also a further issue with some .md files having a permalink, e.g. `_posts/2015-08-03-bugzilla-to-jira.md` to blog/ which means the index files cannot simply be deleted.
I'm not sure how to fix this. My first instinct would be to replace the index.html in the aforementioned directories, rather than adding the new files in the root, but I'm not fully confident.

kathryngo commented:
@djansen-redhat thanks for looking into this! I am not sure what to do either.

kathryngo commented:
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[a discussion](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-KsvjTHDSNC5aORBrS1O:-Kt9VPAsryKHDxy5jSM_:b8722ef) (no related file):*
<details><summary><i>Previously, aeng (Alex Eng) wrote…</i></summary><blockquote>

Which files you're referring to?
</blockquote></details>

_config.yml
_dev_config.yml
Gemfile.lock

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

davidmason commented:
We can easily fit the new pages into the Jekyll structure. It would solve the link issue and remove duplication.

You can set it up to run Jekyll locally, as per the README. I can help convert the hard-coded pages in this PR to use layout templates - it would not take long, it's just pulling out some parts of the static pages into `_layouts`, some conditions in the nav section to highlight the right item, and make sure the internal URLs use the right structure.

djansen-redhat commented:
Thanks @davidmason - I think we will need your help for this one.
I'm currently using the README to test, which was how I found the error I believe. Is it possible with that to make both `url/blog` and `url/blog.html` work?

davidmason commented:
@djansen-redhat I'm not sure about having both those URLs work at once. Would they be pointing to the same content or different? If it is the same then I would update the question to "why do we want both those URLs to work?"

kathryngo commented:
We only need the blog.html file for the moment. Setting up a blog through Jekyll is a seperate issue which we were planning on doing later.

davidmason commented:
@kathryngo we already have a blog set up through Jekyll. We can update `blog/index.html` and `_layouts/page.html` (or switch blog to use a different layout template) if it just needs a visual update.

djansen-redhat commented:
Seems to me we should move e.g. `blog.html` to `blog/index.html` and update the necessary links. We can look at making .md templates later.

kathryngo commented:
@davidmason would rather just push it live by moving it to blog/index.html. This needs to go live asap, do not have the time for templating unless absolutely necessary.

kathryngo commented:
@djansen-redhat: I have moved that blog file and updated the link so it is ready for another test. I am happy to do this testing if you prefer. I just don't know what commands to run.

djansen-redhat commented:
@davidmason @kathryngo done.
Some testing performed, using eyeballs and a linkchecker.

davidmason commented:
Reviewed 1 of 142 files at r2, 20 of 25 files at r9.
Review status: all files reviewed at latest revision, 6 unresolved discussions.

---

*[instances/index.html, line 4 at r9](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-KtF7gvMvFGCDp3zMoc4:-KtF7gvMvFGCDp3zMoc5:b-1twhvv) ([raw file](https://github.com/zanata/zanata.github.io/blob/cb30f0034b3f5c45431f1f6e3a733c130c752531/instances/index.html#L4)):*
> ```HTML
> title: Instances
> layout: newdefault
> description: Instances of Zanata translation sofware
> ```

Change to `Instances of Zanata Platform`

---

*[instances/index.html, line 19 at r9](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-KtF7qnBDYxE2F85uCmH:-KtF7qnBDYxE2F85uCmI:b-nlrik) ([raw file](https://github.com/zanata/zanata.github.io/blob/cb30f0034b3f5c45431f1f6e3a733c130c752531/instances/index.html#L19)):*
> ```HTML
>                     />
>                 </a><br/>
>                 <a href="https://translate.openstack.org">https://translate.openstack.org
> ```

add `</a>`

---

*[instances/index.html, line 30 at r9](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-KtF7nzod2MgUdMfZHvk:-KtF7nzod2MgUdMfZHvl:b-moo982) ([raw file](https://github.com/zanata/zanata.github.io/blob/cb30f0034b3f5c45431f1f6e3a733c130c752531/instances/index.html#L30)):*
> ```HTML
>                 <br/>
>                 <a href="https://fedora.zanata.org">https://fedora.zanata
>                     .org</a>
> ```

join lines

---

*[instances/index.html, line 40 at r9](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-KtF7p923DTYsCc5rG2y:-KtF7p923DTYsCc5rG2z:b-moo982) ([raw file](https://github.com/zanata/zanata.github.io/blob/cb30f0034b3f5c45431f1f6e3a733c130c752531/instances/index.html#L40)):*
> ```HTML
>                 <br/>
>                 <a href="https://translate.jboss.org">https://translate
>                     .jboss.org</a>
> ```

join lines

---

*[instances/index.html, line 50 at r9](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-KtF7uANndwC5otDPZda:-KtF7uANndwC5otDPZdb:b-moo982) ([raw file](https://github.com/zanata/zanata.github.io/blob/cb30f0034b3f5c45431f1f6e3a733c130c752531/instances/index.html#L50)):*
> ```HTML
>                          alt="Zanata"/></a><br/>
>                 <a href="https://translate.zanata.org">https://translate.zanata
>                     .org</a>
> ```

join lines

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35)*
<!-- Sent from Reviewable.io -->

davidmason commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 1 of 1 files at r10.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata.github.io/35#-:-KtFB8HECaRKPUkKxm3c:bnfp4nl)*
<!-- Sent from Reviewable.io -->

kathryngo commented:
Shall look in a few minutes 😄

kathryngo commented:
Looks good. Thanks for fixing this @davidmason and @djansen-redhat. You both are champions.
So are we ready to merge once @aeng reviews that last file?

aeng commented:
Good to merge when @djansen-redhat approved it.
